ui/standard-ui/src/main/java/com/ait/ui/standard/view/HourlyWeatherItemView.kt

```diff
@@ -0,0 +1,31 @@
+package com.ait.ui.standard.view
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.ait.ui.standard.R
+
+class HourlyWeatherItemView(
```

Comment:
Seems like we don't need custom views
Just use inflating in view holders
It's ok if they wont be assigned to ViewModel. We'll do it later

Reply:
Removed views classes and moved the code to view holders